modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
+          timeoutSeconds: 20
+          sideEffects: None
+        webhook.pipeline.tekton.dev:
+          failurePolicy: Fail
+          timeoutSeconds: 20
+          sideEffects: None
+----
++
+You can use similar configuration for webhooks in the `triggers`, `pipelinesAsCode`, and `hub` components.
+
+
+[id="triggers-new-features-1-16_{context}"]
+=== Triggers
+
+* With this update, the `readOnlyRootFilesystem` parameter for the pipelines triggers controller, webhook, Core Interceptor, and event listener is set to `true` by default to improve security and avoid being flagged by the security scanner.
+
+* With this update, you can configure {pipelines-shortname} triggers to run event listeners as a non-root user within the container. To set this option, set the parameters in the `TektonConfig` CR as shown in the following example:
++
+.Example of configuring trigger event listeners to run as non-root
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  trigger:
+    options:
+      disabled: false
+      configMaps:
+        config-defaults-triggers: 
+          data:
+            default-run-as-non-root: "true"
+            default-run-as-user: "65532"
+            default-run-as-group: "65532"
+            default-fs-group: "65532"
+----
++
+Optionally, you can set the values of the `default-run-as-user` and `default-run-as-group` parameters to configure the numeric user ID and group ID for running the event listeners in containers; {pipelines-shortname} sets these values in the pod security context and container security context for event listeners. If you use empty values, the default user ID and group ID of `65532` are used.
++
+You can also set the `default-fs-group` parameter to define the `fsGroup` value for the pod security context, which is the group ID that the container processes use for the file system. If you use an empty value, the default group ID of `65532` is used.
+
+* With this update, in pipelines triggers, the `EventListener` pod template now includes `securityContext` settings. Under these settings, you can configure `seccompProfile`, `runAsUser`, `runAsGroup`, and `fsGroup` parameters when the `el-security-context` flag is set to `true`. 
+
+[id="web-console-new-features-1-16_{context}"]
+=== Web console
+
+* Before this release, when using the web console, you could not see the timestamp for the logs that {pipelines-shortname} created. With this update, the web console includes timestamps for all logs for {pipelines-shortname}.
+
+* With this update, the pipeline run and task run *list* pages in the web console now have a filter for the datasource, that is `k8s` and `TektonResults API`.
+
+* Before this update, when using the web console in the *Developer* perspective, you could not specify the timeout for pipeline runs. With this update, you can now set timeout while starting the pipeline run in the *Developer* perspective of the web console. 
```

Comment:
"you can set a timeout" - "now" does not add value and "timeout" is singular so needs an article